site/pt/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,149 @@
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/9242d13569e9fe67ac01e75d28fa2b6e6bf310d2 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/5715a7b9800532d8b28e2c9fa2d3c1e220ba54a8 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/29236e333a856ae5a952fe4182545b1e2bde5539 #}
+
+Chromium issue: [1208389](https://crbug.com/1208389)
+
+## Depuração CORS aprimorada {: #cors }
+
+Os TypeErrors relacionados ao CORS no **Console** agora estão vinculados ao painel Network e à guia Issues.
+
+Clique nos dois novos ícones ao lado da mensagem de erro relacionada ao CORS para visualizar a solicitação de rede ou entender melhor a mensagem de erro e obter possíveis soluções na guia Problemas.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/VzoUggSoM0FnkDlIFPhq.png", alt="Icons next to the CORS-related error message", width="800", height="485" %}
+
+Chromium issue: [1213393](https://crbug.com/1213393)
+
+## Lighthouse 8.1 {: #lighthouse }
+
+O painel **Lighthouse** agora está executando o Lighthouse 8.1.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/wENi9RXYMxdhm3zI4NVu.png", alt="Lighthouse", width="800", height="628" %}
+
+Se seu site expõe source maps para o Lighthouse, procure o botão **View Treemap** para ver uma análise do JavaScript enviado, filtrável por tamanho e cobertura no carregamento.
+
+O relatório também inclui um novo filtro de métrica (consulte **Mostrar auditorias relevantes para** filtro na captura de tela). Escolha uma métrica para se concentrar nas oportunidades e diagnósticos mais relevantes para melhorar apenas essa métrica.
+
+A **Categoria de desempenho** teve uma série de alterações de pontuação para se alinhar com outras ferramentas de desempenho e refletir melhor o estado da web.
+
+Confira as [notas de lançamento](https://github.com/GoogleChrome/lighthouse/releases) for a full list of changes.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/62b16561e433f4aa1645826923222699ac4bad38 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/16d96a25f24c934ef4dcbbc7b827015abdd228a1 #}
+
+Chromium issue: [772558](https://crbug.com/772558)
+
+## Exibir o novo URL da nota no painel Manifest {: #new-note-url }
+
+O painel Manifest agora exibe o [URL da nova nota](https://wicg.github.io/manifest-incubations/index.html#dfn-note_taking). 
+
+Atualmente no Chrome OS (CrOS), os aplicativos Chrome e aplicativos Android que declaram um recurso de "nova nota" podem ser selecionados como um aplicativo de anotações nas configurações da caneta (aparece se o dispositivo CrOS foi usado com uma caneta). Quando selecionado como um aplicativo de anotações, o aplicativo pode ser iniciado a partir do botão "Criar Anotação" da paleta da caneta. Adicionar o campo `new-note-url` no manifesto do aplicativo faz parte do esforço para adicionar funcionalidade equivalente aos aplicativos da web.
```

Comment:
"Adicionar o campo `new-note-url` no manifesto do aplicativo faz parte do esforço para adicionar funcionalidade equivalente aos aplicativos da web." is sounding a big strange to me.

"Para adicionar a aplicação web como uma das opções de aplicativo de anotação você precisará adicionar o campo `new-note-url` no manifesto web do aplicativo." - assuming that we're referring to the web manifest here. 

However, since the original says "as part of the effort to add the functionality", I'm not sure what else is needed.